docs/codepush.md

```diff
@@ -0,0 +1,122 @@
+# CodePush for React Native support
+
+## Overview
+
+Enable React Native Crash Reporting and Error Tracking to get comprehensive crash reports and error trends with Real User Monitoring.
+
+Each time you release a new [CodePush][1] version for your React Native application, you need to upload the source maps to Datadog to unminify errors.
+
+To achieve this, we recommand using `@datadog/mobile-react-native-code-push` in your app and the [datadog-ci][3] `react-native codepush` command to upload your source maps. It will ensure that the version will be consistent in both reported crashes and uploaded source maps.
```

Comment:
```suggestion
To achieve this, we recommend using `@datadog/mobile-react-native-code-push` in your app and the [datadog-ci][3] `react-native codepush` command to upload your source maps. It will ensure that the version will be consistent in both reported crashes and uploaded source maps.
```